src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,22 @@
 package core.basesyntax;
 
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+
+        if (password == null | repeatPassword == null) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
+
+        if (password.equals(repeatPassword) == false) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
+
+        if (password.length() < 10 | repeatPassword.length() < 10) {
+            throw new PasswordValidationException("Wrong passwords");
+        }
```

Comment:
can we make this checking in the one `if` block?

Reply:
done!